File: 백준/Bronze/10870. 피보나치 수 5/피보나치 수 5.py
import sys

input = sys.stdin.readline


def fib(num):
    # if num == 0:
    #     return 0
    # if num == 1:
    #     return 1
    if num <2:
        return num
    return fib(num - 1) + fib(num - 2)


num = int(input())
print(fib(num))

File: 백준/Bronze/10870. 피보나치 수 5/피보나치 수 5.kt
package org.example

fun fib(num: Int): Int {
    if (num < 2) {
        return num
    }
    return fib(num - 1) + fib(num - 2)
}

fun main() {
    val br = System.`in`.bufferedReader()
    val num = br.readLine().toInt()
    println(fib(num))
    br.close()
}
